frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -149,24 +156,30 @@ const CreateCRDButton: React.FC<CRDButtonProps> = ({ obj, disabled }) => {
   );
 };
 
-const InstallPageCRDRequiredMessage: React.FC<InstallCRDMessageProps> = ({ obj }) => {
-  if (!getCRDFromObject(obj)) {
+const InstallPageInitializationResourceRequiredMessage: React.FC<InstallInitializationResourceMessageProps> = ({
+  obj,
+}) => {
+  if (!getInitializationResourceJSONFromCSV(obj)) {
     return null;
   }
-  let cr = null;
+  let initializationResource = null;
   try {
-    cr = JSON.parse(getCRDFromObject(obj));
+    initializationResource = JSON.parse(getInitializationResourceJSONFromCSV(obj));
   } catch (error) {
     const errorMsg = error.message;
     errorModal({ errorMsg });
   }
-  const crdKind = cr?.[0]?.kind;
-  const crNamespace = cr?.[0]?.metadata?.namespace;
+  const initializationResourceKind = initializationResource?.kind;
+  const initializationResourceNamespace = initializationResource?.metadata?.namespace;
   const description = obj?.metadata?.annotations?.description;
   return (
     <div className="co-clusterserviceversion__box">
       <span className="co-resource-item">
-        <ResourceLink kind={crdKind} name={crdKind} namepace={crNamespace} />
+        <ResourceLink
+          kind={initializationResourceKind}
+          name={initializationResourceKind}
```

Comment:
Not new, but this seems wrong. Why are we passing kind as the `name` prop?

Reply:
For the examples I have, the kind is `StorageCluster` while the name is `portworx` or `example-storagecluster`.  I want the Link to show the kind name.